paddle/phi/api/yaml/op_compat.yaml

```
@@ -2441,6 +2441,15 @@
   outputs :
     out : Out
 
+- op : pull_sparse_v2
+  backward : pull_sparse_v2_grad
+  inputs :
+    { x : Ids, W : w}
+  outputs :
+    out : Out
+  extra :
+    attrs : [int embeddingdim = 11, int tableid = 0, str accessorclass = "", str ctrlabelname = "", int paddingid = 0, bool scalesparsegrad = true, str[] inputnames = {}, bool is_distributed = true]
```

```suggestion
    attrs : [int embeddingdim = 11, int tableid = 0, str accessorclass = "", str ctrlabelname = "", int paddingid = 0, bool scalesparsegrad = true, ‘str[] inputnames = {}’, bool is_distributed = true]
```
看起来是注册的函数签名解析有问题，大概率是这里的问题，可以试下。

---

收到